rmf_demos_maps/maps/clinic/clinic.building.yaml

```diff
@@ -1015,7 +1015,7 @@ levels:
       - [627.881, 1599.344, 0, ""]
       - [627.408, 1497.723, 0, ""]
       - [499.965, 1254.912, 0, ""]
-      - [264.658, 1186.373, 0, L1_sub_waiting_area_1, {is_holding_point: [4, true], is_parking_spot: [4, false]}]
+      - [264.658, 1186.373, 0, L1_sub_waiting_area_1, {is_charger: [4, true], is_holding_point: [4, true], is_parking_spot: [4, true], spawn_robot_name: [1, deliveryRobot_1], spawn_robot_type: [1, DeliveryRobot]}]
```

Comment:
I see a duplicate spawning of `deliveryRobot_1`, which causes the world in `rmf_demos_ign` to fail during start-up

Reply:
Thanks for pointing it out! I removed the duplicate spawn and ignition launches properly now